src/main/java/com/microsoft/graph/http/CoreHttpProvider.java

```diff
@@ -268,7 +268,7 @@ public Request authenticateRequest(Request request) {
 					.tag(RedirectOptions.class, redirectOptions)
 					.tag(RetryOptions.class, retryOptions);
 			
-			String contenttype = null;
+			String contenttype = "";
```

Comment:
We have to pass a contenttype when we do POST without a body. Passing " " is not the correct solution.